pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -740,6 +745,26 @@ func (host *BareMetalHost) OperationMetricForState(operation ProvisioningState)
 	return
 }
 
+// GetImageChecksum returns the hash value and its algo.
+func (host *BareMetalHost) GetImageChecksum() (string, string) {
```

Comment:
If you name the return arguments that will make it clearer to a caller which value is which.